extension/package.json

```diff
@@ -1450,8 +1450,8 @@
           },
           {
             "id": "dvc.learnMore",
-            "title": "Explore More Resources",
```

Comment:
not related change, let's do this separately please

Reply:
Yeah sorry, for a moment I thought it was going to be closely related...